sources/us/wa/statewide.json

```diff
@@ -0,0 +1,26 @@
+{
+    "coverage": {
+        "US Census": {
+            "geoid": "53",
+            "state": "Washington"
+        },
+        "country": "us",
+        "state": "wa"
+    },
+    "data": "http://geoservices.wa.gov/arcgis/rest/services/Parcels/ECY_Washington_State_Parcel_Database_2015/MapServer/0",
```

Comment:
This might need to be:

`"data": "http://geoservices.wa.gov/arcgis/rest/services/Parcels/ECY_Washington_State_Parcel_Database_2016/MapServer/0",`

(note 2016 instead of 2015)